apps/api-extractor/src/utils/TypeScriptMessageSerializer.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as ts from 'typescript';
+
+export class TypeScriptMessageSerializer {
```

Comment:
>Serializer [](start = 30, length = 10)

"Formatter" might be a better term than "serializer", since serialization implies that it's possible to deserialize the result.